server/channels/app/auto_responder_test.go

```diff
@@ -65,16 +71,18 @@ func TestDisableAutoResponder(t *testing.T) {
 	patch.NotifyProps["auto_responder_active"] = "true"
 	patch.NotifyProps["auto_responder_message"] = "Hello, I'm unavailable today."
 
-	th.App.PatchUser(th.Context, user.Id, patch, true)
+	user, appErr := th.App.PatchUser(th.Context, user.Id, patch, true)
+	require.Nil(t, appErr)
 
-	th.App.DisableAutoResponder(th.Context, user.Id, true)
+	err := th.App.DisableAutoResponder(th.Context, user.Id, true)
+	require.Nil(t, err)
 
 	userUpdated1, err := th.App.GetUser(user.Id)
 	require.Nil(t, err)
 	assert.Equal(t, userUpdated1.NotifyProps["auto_responder_active"], "false")
 
-	th.App.DisableAutoResponder(th.Context, user.Id, true)
-
+	err = th.App.DisableAutoResponder(th.Context, user.Id, true)
+	require.Nil(t, err)
 	userUpdated2, err := th.App.GetUser(user.Id)
```

Comment:
Nit:
```suggestion
	require.Nil(t, err)
	userUpdated2, err := th.App.GetUser(user.Id)

```

Reply:
@hanzei So you just want me to add a new empty line after one line number 86??

Reply:
Exactly.